mpxe/protos/adc.proto

```diff
@@ -0,0 +1,5 @@
+syntax = "proto3";
+
+message MxAdcStore {
+  repeated uint32 reading = 2;
```

Comment:
nit: why is this field number 2? I feel like we should just default to using the smallest number (1)